tiledb/sm/rest/curl.cc

```diff
@@ -128,6 +128,14 @@ Status Curl::init(const Config* config) {
     return LOG_STATUS(Status::RestError(
         "Error initializing libcurl; failed to set CURLOPT_ERRORBUFFER"));
 
+  // Ignore ssl validation if the user has set rest.ignore_ssl_validation = true
+  const char* ignore_ssl_validation = nullptr;
+  RETURN_NOT_OK(config_->get("rest.ignore_ssl_validation", &ignore_ssl_validation))
+  if (ignore_ssl_validation != nullptr && !std::strcmp(ignore_ssl_validation, "true")) {
```

Comment:
You can use https://github.com/TileDB-Inc/TileDB/blob/dev/tiledb/sm/misc/utils.cc#L171 to convert string to bool if you want, it also handles case normalization.